docs/source/nlp/tutorials/quickstarts/plot_token_classification.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__token_classification_quickstart:
+
+Test NLP Token Classification Tasks - Quickstart
+************************************************
+
+In this quickstart guide, we will give a brief example of using the deepchecks NLP package to analyze and evaluate token
+classification tasks. For a more complete example showcasing the range of checks and capabilities of the NLP package,
+refer to our :ref:`Multiclass Quickstart <nlp__multiclass_quickstart>`. We will cover the following steps:
+
+1. `Creating a TextData object and auto calculating properties <#setting-up>`__
+2. `Running checks <#running-checks>`__
+
+To run deepchecks for NLP, you need the following for both your train and test data:
+
+1. Your :ref:`text data <nlp__textdata_object>` - a list of strings, each string is a single sample
+   (can be a sentence, paragraph, document, etc.).
+2. Your labels - either a :ref:`Text Classification <nlp_supported_text_classification>` label or a
+   :ref:`Token Classification <nlp_supported_token_classification>` label. These are not needed for checks that
+   don't require labels (such as the Embeddings Drift check or most data integrity checks), but are needed for
```

Comment:
ref to check?

Reply:
included later when check is actually shown